libs/api/libbpf_program.cpp

```diff
@@ -261,13 +256,25 @@ bpf_object__unpin_programs(struct bpf_object* obj, const char* path)
     return 0;
 }
 
+void
+bpf_link__disconnect(struct bpf_link* link)
```

Comment:
move this to libbpf_link.cpp.  (bpf_link__destroy should move there too)

Reply:
Done.